src/lib/Microsoft.Fx.Portability.Reports.Excel/ExcelOpenXmlOutputWriter.cs

```diff
@@ -42,27 +42,16 @@ internal static class DetailsPage
 
         // This is the second item added in AddStylesheet.  Update accordingly
         private const int _hyperlinkFontIndex = 1;
+
         private readonly ITargetMapper _mapper;
+        private readonly AnalyzeResponse _response;
         private readonly string _description;
-        private readonly ReportingResult _analysisReport;
-        private readonly IEnumerable<BreakingChangeDependency> _breakingChanges;
-        private readonly DateTimeOffset _catalogLastUpdated;
-        private readonly IList<ExceptionInfo> _throwingMembers;
-
-        public ExcelOpenXmlOutputWriter(
-            ITargetMapper mapper,
-            ReportingResult analysisReport,
-            IEnumerable<BreakingChangeDependency> breakingChanges,
-            DateTimeOffset catalogLastUpdated,
-            IList<ExceptionInfo> throwingMembers = null,
-            string description = null)
+
+        public ExcelOpenXmlOutputWriter(ITargetMapper mapper, AnalyzeResponse response, string description)
```

Comment:
Sure, it can be large, but it's a reference so this is actually more performant to just pass a single reference. I can't imagine any memory issue from this - let me know if there's something you see otherwise